### ISSUE TITLE
Clone historic 2018 main Disability rates (create in spreadsheet) 

### ISSUE THREAD

**Issue by AnneHurley** [department-of-veterans-affairs/va.gov-team#10893]
Tasks:

- [X] Disability main rates, create 2018 spreadsheet request of Elana as dry run for future rates tables. Request sent 6/15 by Anne; reminder sent by Beth 7/3  STATUS: Blocked
- [X] Turn finished spreadsheets into .CSV files. Anne
- [X] Upload .CSV files into Drupal to create  actual historic 2018 rates page. Selina
- [x] Copyedit and validate. Randi
- [x] Share with stakeholders for review. Beth
- [x] Add link in left nav. Randi
- [x] Publish/ Randi
- [ ] Redirect legacy page to this page as a batch. Selina. 

Epic: #10435
Preview:

**Comment by AnneHurley** [comment]
@MickinSahni  We're pulling this individual ticket  out from the 2018 historic rates epic, as we are still waiting on stakeholder Elana to create the spreadsheet with those rates (which I'll turn into CSV files and upload). 

@bethpottsVADEPO  is in contact with Elana/VBA on the status of this.

**Comment by RLHecht** [comment]
Per Beth, the stakeholder is going to return the spreadsheet by the end of this week so we can start work on this for the coming sprint. FYSA @MickinSahni

**Comment by bethpottsVADEPO** [comment]
Here's the spreadsheet from the stakeholder with the rates info inserted for 2018. @RLHecht let me know how it goes uploading using the CSV process. [https://app.zenhub.com/files/133843125/d84bb95b-8322-40fb-998f-318fdafcd5ce/download](https://app.zenhub.com/files/133843125/d84bb95b-8322-40fb-998f-318fdafcd5ce/download)

**Comment by lunascoop** [comment]
Hi @RLHecht the 2018 rates page is ready for review. Here's the link: http://preview-prod.vfs.va.gov/preview?nodeId=6793

**Comment by RLHecht** [comment]
@lunascoop This looks great. @bethpottsVADEPO should we change how we do the "all rates in U.S. $" on this page too? Put "(in U.S. $)" in each column heading. I think we should for consistency but it would just mean we have it in lots of column headers. Too busy?

**Comment by bethpottsVADEPO** [comment]
@RLHecht I agree that for consistency we should bring the (in U.S. $) into the chart itself, so it appears just above the dollar amounts. What I'd like to know (I guess this is a Kevin Q) is whether it's possible to use tags to achieve this:
30%<br>
disability rating<br>
monthly payment<br>
(in U.S. $)

Is there a way to make the first column skinnier perhaps by adding a break tag after "no" in the second dependent status row (With spouse), so that it renders as:

**With spouse** (no<br>
parents or children)

That would make the column slimmer and hopefully allow us to add a little width to each of the other columns. If this doesn't work, is there any other creative way to do it along those lines of using break tags or something else?

If we can achieve all this, the headers on this page would become:
Basic rates<p>
Added amounts

**Comment by RLHecht** [comment]
@bethpottsVADEPO Trying to set up the table columns as you had above wasn't working. Keeping "disability rating" and "monthly payment" on their own lines made the columns too wide and didn't show the whole table on the page. I just added in the "(in U.S. $)" part and kept that on one line. Take a look and let me know what you think. If it's good this is ready to send to the stakeholder. http://preview-prod.vfs.va.gov/preview?nodeId=6793

**Comment by bethpottsVADEPO** [comment]
This works, thank you! We can always iterate down the road if we want to. I'll share with Elana.

**Comment by bethpottsVADEPO** [comment]
@RLHecht I forwarded you the email from Elana. She wants her supervisor to see this page before we go live with it. So hold on publishing for now. Thanks!

**Comment by bethpottsVADEPO** [comment]
@RLHecht OK, we just got the thumbs up! You can publish.

**Comment by RLHecht** [comment]
Just published! Moving to validate and will close after this afternoon's deploy. @lunascoop Now that all the 2018 disability rates pages are up, you should be able to set up redirects for them all in another ticket if there isn't one already.

**Comment by RLHecht** [comment]
Validated on live site. Closing ticket. FYSA @MickinSahni @bethpottsVADEPO
